src/schema/sale/index.js

```diff
@@ -293,6 +294,10 @@ export const SaleType = new GraphQLObjectType({
           }
         },
       },
+      partner: {
+        type: Partner.type,
+        resolve: ({ partner }) => partner,
```

Comment:
This will only return in the `partner` data that gravity sends back in the `sale` JSON (which may not include all of the fields we need). You probably want to actually load the partner here, similar to what we do for the `promoted_sale` below.

Reply:
That works okay for me since all I need is the partner's `_id`. Do you think it's worth the extra request to fetch a more complete record of the partner?

Reply:
I think it's fine to start from here and expand when needed.

Reply:
Makes sense! Can we possibly add a comment? 😄 This is probably going to be fine for a while, but (as you bring up @williardx ) it's sort of an odd experience from a graphiQL UI or otherwise that suggests you can retrieve fields that won't exist.

Reply:
Will do!